navigation/navigation-runtime/src/jsMain/kotlin/androidx/navigation/BrowserHistory.js.kt

```diff
@@ -41,11 +41,14 @@ import org.w3c.dom.Window
  * @param navController The [NavController] instance to bind to browser window navigation.
  * @param getBackStackEntryRoute An optional function that returns the route to show for a given [NavBackStackEntry].
  */
+@Suppress("UnusedReceiverParameter")
+@Deprecated(message = "Use bindToBrowserNavigation", replaceWith = ReplaceWith("navController.bindToBrowserNavigation(getBackStackEntryRoute)"))
 @ExperimentalBrowserHistoryApi
```

Comment:
Well, we don't need to keep it forever for experimental API, especially if we didn't release a stable version with it.

Please create a task for 1.10 to remove it

Reply:
https://youtrack.jetbrains.com/issue/CMP-8434